py/audience_creation_example.py

```diff
@@ -0,0 +1,99 @@
+'''
+    Filename: audience_creation_example.py
+    Author: Brett Coker
+    Python Version: 3.6.1
+
+    An example of how to use the Lotame API to create a basic audience,
+    consisting of two behaviors ANDed together. This script assumes that the
+    audience should be created on Enrich, with the APR disabled.
+'''
+import requests
+import sys
+from getpass import getpass
+
+api_url = 'https://api.lotame.com/2/'
+auth_url = 'https://crowdcontrol.lotame.com/auth/v1/tickets'
+
+
+def main():
+    username = input('Username: ')
+    password = getpass()
+    payload = {'username': username, 'password': password}
+
+    # Exit if we cannot get the ticket-granting ticket (i.e. if the username
+    # and/or password are incorrect)
+    try:
+        tgt = requests.post(auth_url, data=payload).headers['location']
+    except KeyError:
+        print('Error: Invalid username and/or password.')
+        sys.exit()
+
+    client_id = input('Client ID: ')
+    audience_name = input('New Audience Name: ')
+
+    # Put the two behavior IDs in a list for future iteration
+    behavior_definitions = []
+    behavior_definitions.append(input('First Behavior ID: '))
+    behavior_definitions.append(input('Second Behavior ID: '))
+
+    behaviors = []
+    first_behavior = True
+    for behavior_id in behavior_definitions:
```

Comment:
Oops. Sorry about missing confusing which was which. I thought behavior_definitions made sense to contain the actual json for the behavior (the return from create_behavior_definition call). The one that just contains ids probably should just be behavior_ids or something similar.